cmd/helm/search_repo.go

```diff
@@ -50,7 +50,7 @@ Examples:
     # Search for release versions matching the keyword "nginx", including pre-release versions
     $ helm search repo nginx --devel
 
-    # Search for the latest patch release for nginx-ingress 1.x
+	# Search for the latest stable release for nginx-ingress with a major version of 1
```

Comment:
can you switch this back to using spaces? Help string text should use spaces as indentation to ensure it's displayed consistently in the terminal.

Reply:
oops. great catch. fixed.